docs/endpoints/post-token-validate.md

```diff
@@ -30,18 +30,18 @@ Used by: This endpoint is used mainly by publishers.
-- To test identities, use the `validate@email.com` email address or `+12345678901` phone number. For details, see the FAQs sections in the [UID2 SDK for JavaScript Integration Guide](../guides/publisher-client-side.md) and [Server-Only Integration Guide](../guides/custom-publisher-integration.md) for publishers.
 
-| Body Parameter | Data Type | Attribute | Description |
-| :--- | :--- | :--- | :--- |
-| `token` | string | Required | The advertising token returned by the [POST /token/generate](post-token-generate.md) response. |
-| `email` | string | Conditionally Required |  The email address for token validation. |
-| `email_hash` | string | Conditionally Required | The [Base64-encoded SHA-256](../getting-started/gs-normalization-encoding#email-address-hash-encoding) hash of a [normalized](../getting-started/gs-normalization-encoding#email-address-normalization) email address for token validation. |
-| `phone` | string | Conditionally Required | The [normalized](../getting-started/gs-normalization-encoding#phone-number-normalization) phone number for which to generate tokens. |
-| `phone_hash` | string | Conditionally Required | The [Base64-encoded SHA-256](../getting-started/gs-normalization-encoding#phone-number-hash-encoding) hash of a [normalized](../getting-started/gs-normalization-encoding#phone-number-normalization) phone number. |
+For details, see [FAQs for Publishers Using an SDK](../getting-started/gs-faqs.md#faqs-for-publishers-using-an-sdk) or [FAQs for Publishers Not Using an SDK](../getting-started/gs-faqs.md#faqs-for-publishers-not-using-an-sdk).
 
+| Body Parameter | Data Type | Attribute | Description | Value |
+| :--- | :--- | :--- | :--- | :--- |
+| `token` | string | Required | The advertising token returned by the [POST /token/generate](post-token-generate.md) response. |
+| `email` | string | Conditionally Required |  The email address for token validation.<br/>The only valid value is: `validate@email.com`. |
+| `email_hash` | string | Conditionally Required | The [Base64-encoded SHA-256](../getting-started/gs-normalization-encoding#email-address-hash-encoding) hash of the [normalized](../getting-started/gs-normalization-encoding#email-address-normalization) email address for token validation (`validate@email.com`).<br/>The only valid value is: `LdhtUlMQ58ZZy5YUqGPRQw5xUMS5dXG5ocJHYJHbAKI=`. |
+| `phone` | string | Conditionally Required | The [normalized](../getting-started/gs-normalization-encoding#phone-number-normalization) phone number for which to generate tokens.<br/>The only valid value is: `+12345678901`. |
+| `phone_hash` | string | Conditionally Required | The [Base64-encoded SHA-256](../getting-started/gs-normalization-encoding#phone-number-hash-encoding) hash of a [normalized](../getting-started/gs-normalization-encoding#phone-number-normalization) phone number.<br/>The only valid value is: `EObwtHBUqDNZR33LNSMdtt5cafsYFuGmuY4ZLenlue4=`. |
 
 ### Request Examples
 
```

Comment:
I think we were going to change email here?

Reply:
Updated the email example.